### ISSUE TITLE
HW4: Contrast should be 3:1

### ISSUE THREAD

**Issue by iane884** [NU-CS-Software-Studio-Spring-26/project-next-step-assistant#23]
elements must have sufficient contrast between text in the foreground and background colors behind it[/+]
[+]in accordance with WCAG 2 AA contrast ratio thresholds.
